types/apis/orders.d.ts

```diff
@@ -135,37 +135,39 @@ export type ShippingInfoOption = {
     selected: boolean;
 };
 
-export type ShippingInfo = {
+export interface ShippingInfoBase {
```

Comment:
we could stop exporting this type in favor of just `ShippingInfo`

Reply:
Great point! Updated.